<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_main</class>
 <widget class="QWidget" name="client_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华康方圆体W7</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>顾客主页面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sdufe.qrc">
    <normaloff>:/山财大校徽.png</normaloff>:/山财大校徽.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看近期热映</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>办理会员卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>110</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>查看个人编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>110</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>说明：若用户名只有英文或数字，直接输入查询即可</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>若用户名中含有中文字符，应用英文输入法双引号括起来再查询，例如&quot;张三&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>此外，记住您的顾客编号，它将非常重要！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>若想修改个人信息，直接在表中修改即可，然后点击此按钮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sdufe.qrc"/>
 </resources>
 <connections/>
</ui>
